build-wheelhouse/action.yml

```
@@ -134,7 +134,7 @@ runs:
     - name: "Generate the wheelhouse"
       shell: bash
       run: |
-        python -m pip wheel . -w wheelhouse
+        python -m pip wheel ${{ env.install_target }} -w wheelhouse
```

This ensures that if no optional deps are declared, the project gets installed. Also, since this is an env var, it may be better to use upper case:

```suggestion
        python -m pip wheel .[${{ env.OPTIONAL_DEPENDENCIES }}] -w wheelhouse
```

Note that developers can pass multiple optional deps, like "OPTIONAL_DEPENDENCIES: 'doc,tests'".

---

@jorgepiloto - this is handled here already

https://github.com/ansys/actions/blob/7988e4d1644a4077e88eaa92fe99d5dfb41c6e99/build-wheelhouse/action.yml#L84-L89

---

It's not a real env var, that's why I defined it in small caps in the past. It's coming from an input.

---

Oh! My fault. Approving this.